frontend/actions/apiActions.js

```diff
@@ -22,22 +23,36 @@ export function loadApis () {
       })
       .getConfig()
       .then(config => {
-        apiBridgeClient({api}).getCollections().then(({collections}) => {
+        if ('undefined' === typeof config) {
```

Comment:
For readability I'd reverse the statement:
```javascript
if (typeof config === 'undefined') {
...
}
```
You could also consider using the `undefined` primitive type, which we've previously used in the app.
```javascript
if (config === undefined) {
...
}
```

Reply:
I'm used to doing static-first, to prevent accidental ='ing when I mean to == or ===.
But if this project requires the opposite I could change things...

Reply:
As a rule, we always use `===`, although worth checking with @eduardoboucas incase this has changed. I would definitely reverse this for readability and consistency with the rest of the code :)